core/uwsgi.c

```diff
@@ -3338,14 +3338,6 @@ int uwsgi_start(void *v_argv) {
 		}
 	}
 
-	// master fixup
```

Comment:
you should leave this part in for not breaking other plugins

Reply:
@unbit I did not see the `master_fixup` hook used elsewhere in the repository but that doesn't mean it's not used by a plugin a user might have, so I can add this back. Are you aware of such a plugin?